cmd/main.go

```diff
@@ -70,7 +72,40 @@ func toExitCode(codeStr string) int {
 	return code
 }
 
-func parseCliArgs(flags *args) (*cobra.Command, error) {
+func boolPtr(b bool) *bool {
+	newBool := b
+	return &newBool
+}
+
+func strPtr(str string) *string {
+	s := str
+	return &s
+}
+
+func strNilOrDefault(str *string, dflt string) string {
+	if str != nil {
+		return *str
+	}
+	return dflt
+}
+
+func boolNilOrDefault(b *bool, dflt bool) bool {
+	if b != nil {
+		return *b
+	}
+	return dflt
+}
+
+func parseCliArgs() (*cobra.Command, error) {
+	flags := args{Config: gitopszombiesv1.Config{
+		TypeMeta:         metav1.TypeMeta{},
+		ExcludeClusters:  nil,
+		ExcludeResources: nil,
+		Fail:             boolPtr(false),
+		IncludeAll:       boolPtr(false),
+		LabelSelector:    strPtr(""),
+		NoStream:         boolPtr(false),
+	}}
```

Comment:
Can't we just have values in instead pointers in the v1.Config api? I reckon that is the nicer approach and you would not need to initialize the config that way.